commit 202b7c1261df1bb39b93ec5901ce90303bd31239 (refs/changes/26/726/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-18T10:04:59-07:00 (3 years ago)
    
    RELENG-728 purge-mg-builds: support verbose mrm
    Reviewed by: Robert Mustacchi <rm@joyent.com>

diff --git a/tools/purge-mg-builds b/tools/purge-mg-builds
index 9839083..0a6a5f9 100755
--- a/tools/purge-mg-builds
+++ b/tools/purge-mg-builds
@@ -233,7 +233,11 @@ function purge_dir
     dir="$1"
     log "mrm -r $dir"
     if [[ "$opt_dryrun" == "no" ]]; then
-        mrm -r "$dir"
+        if [[ -n "$TRACE" ]]; then
+            mrm -rv "$dir"
+        else
+            mrm -r "$dir"
+        fi
     fi
 }
 
